wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-stats.php

```diff
@@ -162,9 +176,230 @@ public function __invoke( bool $echo_the_values = false ) {
+
+		restore_current_blog();
+
+		return $locale_requests;
+	}
+
+	/**
+	 * Fetch total number of editor requests and number of unresolved editor requests.
+	 *
+	 * @return array $editor_requests Array of the count of each of resolved and unresolved editor requests.
+	 */
+	private function get_editor_requests() {
+		$editor_requests = array();
+		switch_to_blog( MAKE_POLYGLOTS_BLOG_ID );
+		register_taxonomy(
+			'p2_resolved',
+			'post',
+			array(
+				'public'    => true,
+				'query_var' => 'resolved',
+				'rewrite'   => false,
+				'show_ui'   => false,
+			)
+		);
+		$args                     = array(
+			'post_type'   => 'post',
+			'tag'         => 'editor-requests',
+			'post_status' => 'publish',
+		);
+		$editor_requests['total'] = ( new WP_query( $args ) )->found_posts;
+		$args                     = array(
+			'post_type' => 'post',
+			'tag'       => 'editor-requests',
+			'tax_query' => array(
+				array(
+					'taxonomy' => 'p2_resolved',
+					'field'    => 'slug',
+					'terms'    => 'unresolved',
+				),
+			),
+		);
+		$editor_requests['unresolved_editor_requests'] = ( new WP_query( $args ) )->found_posts;
+		restore_current_blog();
+
+		return $editor_requests;
+	}
+
+	/**
+	 * Fetch stats data for a specific date.
+	 *
+	 * @param string $date The date for the record to be retrieved.
+	 *
+	 * @return mixed
+	 */
+	private function get_data_for_date( $date ) {
+		global $wpdb;
+		$sql     = "SELECT * FROM polyglot_stats WHERE date = '" . $date . "'";
+		$results = $wpdb->get_results( $sql );
+		return $results[0];
+	}
+
+
+	/**
+	 * Get the percentage of non-EN_US sites.
+	 *
+	 * @return float
+	 */
+	private function get_translated_sites_pct() {
+		$translation_stats_json  = file_get_contents( 'https://api.wordpress.org/stats/locale/1.0/' );
+		$ranslation_stats_array  = $translation_stats_json && '{' == $translation_stats_json[0] ? json_decode( $translation_stats_json, true ) : null;
+		$wp_translated_sites_pct = 100 - $ranslation_stats_array['English (US)'];
+		return $wp_translated_sites_pct;
+	}
+
+	/**
+	 * Print stats compared week on week.
+	 *
+	 * @param string $current_date The date for which we display the stats.
+	 * @param string $old_date The date to compare the stats with.
+	 *
+	 * @return void
+	 */
+	private function print_stats_comparison( $current_date, $old_date = null ) {
+		if ( ! $this->is_date_valid( $current_date ) || ( $old_date && ! $this->is_date_valid( $old_date ) ) ) {
+			return;
+		}
+		$old_date = is_null( $old_date ) ? gmdate( 'Y-m-d', strtotime( '-1 week' ) ) : $old_date;
+
+		$current_date_data = $this->get_data_for_date( $current_date );
+		$old_date_data     = $this->get_data_for_date( $old_date );
+
+		if ( ! $current_date_data || ! $old_date_data ) {
+			return;
+		}
+		$stats_diff = array();
+		foreach ( $current_date_data as $key => $value ) {
+			$stats_diff[ $key ] = $value - $old_date_data->$key;
```

Comment:
I think it'd be better for consistency to also put this into an object.
```suggestion
			$stats_diff->$key = $value - $old_date_data->$key;
```